tools/lkl/lib/hijack/hijack.c

```diff
@@ -291,7 +304,25 @@ int select(int nfds, fd_set *r, fd_set *w, fd_set *e, struct timeval *t)
 	return lkl_call(__lkl__NR_select, 5, nfds, r, w, e, t);
 }
 
-HOOK_CALL_USE_HOST_BEFORE_START(epoll_create);
+HOST_CALL(epoll_create);
+int epoll_create(int flags)
```

Comment:
thanks for the explanation.

`HOOK_CALL_USE_HOST_BEFORE_START` was needed before when we used `epoll_create` in virtio_net_fd.c.  Nowadays we don't use and it should be safe now.